Formula/emscripten.rb

```diff
@@ -3,18 +3,21 @@ class Emscripten < Formula
   homepage "https://kripken.github.io/emscripten-site/"
 
   stable do
-    url "https://github.com/kripken/emscripten/archive/1.36.13.tar.gz"
-    sha256 "5e5370384a80775eaed63f38dcbc11f38620239f21c3fd6ba3725b5c982a2736"
+    version "1.36.14"
+    url "https://github.com/kripken/emscripten/archive/#{version}.tar.gz"
+    sha256 "89febe6c56c36ded3a6323d40342196d961eb1a7878b32912a649734962cb5ee"
 
     resource "fastcomp" do
-      url "https://github.com/kripken/emscripten-fastcomp/archive/1.36.5.tar.gz"
-      sha256 "322501d14eb90b5590d463ef2ae1b358c07c590440d7bd21b60ea88885bc2fa0"
+      url "https://github.com/kripken/emscripten-fastcomp/archive/#{version}.tar.gz"
```

Comment:
Sorry, I forgot this doesn't work as expected. You can either take the approach from `swift.rb` or (preferred) write something in the `test do` that will reproduce the failure you were seeing when the versions were mismatched. That will stop this being merged with that error.